src/Controller/Action/RenderHeaderTemplateAction.php

```diff
@@ -10,29 +10,29 @@
 
 namespace BitBag\SyliusWishlistPlugin\Controller\Action;
 
-use BitBag\SyliusWishlistPlugin\Context\WishlistContextInterface;
+use BitBag\SyliusWishlistPlugin\Repository\WishlistRepositoryInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Twig\Environment;
 
 final class RenderHeaderTemplateAction
```

Comment:
What's the purpose of this?

Reply:
Bcs when after creating a new wishlist (more than one) widget need to has route for list of wislists, no one wishlist with checkout products.